backend/elasticsearch-aws/src/main/java/module-info.yml

```diff
@@ -0,0 +1,32 @@
+name: org.hibernate.search.backend.elasticsearch.aws
+
+requires:
+  - module: org.apache.httpcomponents.httpasyncclient
+  - module: org.apache.httpcomponents.httpclient
+  - module: software.amazon.awssdk.regions
+
+  - module: org.apache.httpcomponents.httpasyncclient
+  - module: org.apache.httpcomponents.httpclient
+  - module: org.apache.httpcomponents.httpcore
+  - module: software.amazon.awssdk.regions
+  - module: software.amazon.awssdk.auth
+  - module: software.amazon.awssdk.http
+  - module: org.jboss.logging
```

Comment:
Wait, I though this new plugin could auto-generate that kind of things, like moditect does... did I misunderstand?

Having to maintain these lists manually would be a big step back from what we had with moditect... Ideally we'd want the list automatically generated, with the option to amend it for false-positives/false-negatives.

Reply:
>  // todo: detect requires

^ on autodetecting the requires 😃. So I suppose we could try to add that in the plugin ...

Reply:
I suppose we could, but if we end up working a lot to get this new plugin up to speed with moditect, then we might need to question whether it's really worth it... let's discuss this on Zulip, you are best-placed to judge of that.